front/ui/ui-charts/src/speedSpaceChart/components/SpeedSpaceChart.tsx

```diff
@@ -142,6 +146,18 @@ const SpeedSpaceChart = ({
       }}
       tabIndex={0}
     >
+      <div className="speed-space-h-slider-container">
+        <Slider
+          className="speed-space-h-slider"
+          width={ZOOM_CONFIG.SLIDER_WIDTH}
+          min={ZOOM_CONFIG.MIN_RATIO}
+          value={store.ratioX < ZOOM_CONFIG.MIN_RATIO ? ZOOM_CONFIG.MIN_RATIO : store.ratioX}
+          onChange={(e) => {
+            const value = Math.max(ZOOM_CONFIG.MIN_RATIO, Number(e.target.value));
+            setZoomLevel(value, setStore);
```

Comment:
Hum can you compute the leftOffset value in your helper function and set the store here ? It's better to keep the update of the state in the component which contains it.

```suggestion
            setStore((prev) => ({
              ...prev,
              ratioX: value,
              leftOffset: computeLeftOffsetOnZoom(value) // corresponds to your current setZoomLevel
            }));
```